website/docs/integrations/jira-server-plugin-installation.md

```diff
@@ -0,0 +1,86 @@
+---
+id: jira_server_plugin_installation
+title: Jira Server Integration - Installation
+---
+
+> The Jira Server Plugin is part of Unleash Enterprise.
+
+With the Unleash Jira server plugin you can create new feature toggles in Unleash from a Jira issue as well as link
+existing toggles to Jira issues.
+
+The plugin also shows you current status of connected toggles.
+
+## Required access levels
+
+### Unleash
+
+You will need an Unleash Instance admin to configure an admin API token.
```

Comment:
Should 'Instance' be capitalized?

```suggestion
You will need an Unleash instance admin to configure an admin API token.
```

Reply:
Actually reads better without it at all - I'm yanking it out